urihandler/handler.py

```diff
@@ -37,7 +37,8 @@ def handle(self, uri, request):
                     redirect = self._get_redirect_based_on_accept_header(
                         request.accept, redirect
                     )
-                redirect = redirect.format(**m.groupdict())
+                unmatched_part = (uri.replace(m.group(), ""))
+                redirect = redirect.format(**m.groupdict()) + unmatched_part
```

Comment:
Ik zou gwn `request.query_string` doen ipv unmatched part. Het `?` zit er wel niet bij.

Reply:
dat is leeg bij mij, de query paramaters zijn onderdeel van de bevraagde uri, geen echte query paramaters

Reply:
denk dat die params wel altijd best url encoded doorkomen dat het als onderdeel wordt gezien vd uri anders werkt het niet

Reply:
In de praktijk zijn alle urls wel gedefinieert met `$` achteraan. Dus de match zal falen daarmee.
https://github.com/OnroerendErfgoed/scripts/blob/master/servers/joeri/config/urihandler.yaml

De queryparams zullen eraf moeten voor de match. Ik denk dat simpele string manipulatie wel ok is. Gewoon alles na het eerste vraagteken eraf, en bijhouden om er later terug aan te plakken.